diff --git a/datepicker/index.mjs b/datepicker/index.mjs
index 2ca2b5c5aece09616643f0c2c5b44715602373d2..fac790030293bbaf839e4ce60ef9363e464d3cd6 100644
--- a/datepicker/index.mjs
+++ b/datepicker/index.mjs
@@ -493,7 +493,7 @@ var script = {
         var end = this.modelValue[1] ? this.modelValue[1].getFullYear() : null;
         return start === year || end === year || start < year && end > year;
       } else {
-        return value.getFullYear() === year;
+        return this.modelValue.getFullYear() === year;
       }
     },
     isDateEquals: function isDateEquals(value, dateMeta) {
